build_all.sh

```diff
@@ -1,52 +1,22 @@
 #! /bin/sh
 
 # build development version of the compiler; can be rerun safely.
-# arguments can be passed, e.g. `--os freebsd`
+# arguments can be passed, e.g.:
+# CC=gcc ucpu=amd64 uos=darwin
+#
+# when `NIM_CSOURCES_USE_BUILD=1` environment is set, it will build csourcesAny with `build.sh`
```

Comment:
What? Why? Remove this. We run `build_all.sh` and it does what it does, no inference from environment pixies please.

Reply:
done via https://github.com/nim-lang/Nim/pull/17815#discussion_r618996796